src/spacel/aws/elb.py

```diff
@@ -18,6 +18,9 @@ def health(self, manifest):
             logger.debug('ELB not configured.')
             return True
 
+        # make sure the instance is registered to our ELB first
+        self._register_instance_elb(elb)
```

Comment:
Does https://github.com/pebble/spacel-agent/blob/73c891f3760a5dfb69027b96f8ca003fe7e9d0b0/src/spacel/aws/elb.py#L28 throw a useful `ClientError` if we're not in the ELB?

Maybe instead of registering _every_ time (which for >90% will be a NOOP?), we can manually register+retry _after_ the `DescribeInstanceHealth` call has returned a `that instance isn't even registered` response.